core/src/main/java/cn/edu/tsinghua/iot/benchmark/mode/BaseMode.java

```diff
@@ -62,7 +62,17 @@ public abstract class BaseMode {
   protected Measurement measurement = new Measurement();
   protected long start = 0;
 
-  protected abstract boolean preCheck();
+  protected boolean preCheck() {
+    List<DBConfig> dbConfigs = new ArrayList<>();
+    dbConfigs.add(config.getDbConfig());
+    if (config.isIS_DOUBLE_WRITE()) {
+      dbConfigs.add(config.getANOTHER_DBConfig());
+    }
+    if (config.isIS_DELETE_DATA() && (!cleanUpData(dbConfigs, measurement))) {
+      return false;
+    }
+    return !config.isCREATE_SCHEMA() || (registerSchema(measurement));
+  }
```

Comment:
Consider whether these options need to be done before all mode

Reply:
I am also confused about whether to make modifications here. Two classes extend base class and  use nearly identical code for `preCheck`, while the other two employ different approaches for `preCheck`. I can revert this change